### PR TITLE
PLF-8376 update csrf check method

### PR DESCRIPTION
**Pull request by boubaker**
The csrf check method has been modified in https://github.com/exodev/gatein-portal/pull/236 to allow passing the HTTP Request parameter switch request context: WebUI or REST. Thus this PR applies those changes on the static method of CSRFTokenUtil.getToken

### CODE REVIEW

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIPreferencesForm.java [hunk 1]
```diff
@@ -46,6 +46,8 @@
 import org.exoplatform.webui.form.UIFormSelectBox;
 import org.exoplatform.webui.form.input.UICheckBoxInput;
 
+import japa.parser.ast.expr.ThisExpr;
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIPreferencesForm.java]: Useless import

Comment (boubaker) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIPreferencesForm.java]: removed

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIPreferencesForm.java [hunk 2]
```diff
@@ -185,7 +185,7 @@ public void begin() throws Exception {
           .append(ACTION)
           .append("\" value=\"\"/>");
     writer.append("<input type=\"hidden\" name=\"").append(CSRFTokenUtil.CSRF_TOKEN).append("\" value=\"");
-    writer.append(CSRFTokenUtil.getToken());
+    writer.append(CSRFTokenUtil.getToken(getRequest()));
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIPreferencesForm.java]: It should use the new method created in https://github.com/exodev/gatein-portal/pull/236/ : org.exoplatform.webui.Util.getRequest()

Comment (boubaker) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIPreferencesForm.java]: I think that @waelghannem was preparing it :-)
See his commit here:
https://github.com/exodev/gatein-portal/pull/236/commits/f153f36bd1d9674bd900cc3e79796e63fa197f7c

**Pull request merged**